traffic-sniffer/lib/request-metadata/file-config.ts

```diff
@@ -0,0 +1,118 @@
+import fs from "fs";
+import { SnifferConfig } from "./sniffer/sniffer";
+
+const setupFilePath =
+  process.env.SETUP_FILE_PATH ?? "./config/sniffers-setup.json";
```

Comment:
Can we add sample `sniffers-setup.json` to read from??
Otherwise I think when other pull, it would fail without .env file as well?

Reply:
This file will be created if not found, look at validateSetupFileExists function

Reply:
Adding a sample config is a good Idea. 
Also we could use the sharkio localhost:5173 backend for a demo config. @shravan20

Reply:
I can create a sample file and it's `npm run` command could add the env var with its path

Reply:
> Adding a sample config in the project for reference is a good Idea.
Also we could use the sharkio localhost:5173 backend for a demo config

We can pick it up as separate issue if required. Lets add more details and templates for each of configs that are required and document them

Reply:
opened an issue
https://github.com/idodav/sharkio/issues/84